articles/anomaly-detection/references/brute-force-protection-triggers-actions.md

```diff
@@ -51,12 +51,10 @@ If this block is triggered, additional access attempts are released one-at-a-tim
 
 ### Remove block
 
-Auth0 emails the dashboard administrator(s) when this block is triggered. Within this email there's a link the owner can click on to clear the block.
+Auth0 emails the dashboard administrator(s) when this block is triggered. The email contains a link that the owner can click to navigate to tenant logs to examine which IPs have been blocked. There is also a link to documentation that explains how to remove the block.
```

Comment:
It seems really strange to me to have a Remove block section that basically says look at the email to learn how to remove the block. I'd suggest we include a here the steps to unblock: 
1. check your logs - we could include an example query here of either `type:limit_mu`
2. use the api - specifically pointing them to https://auth0.com/docs/api/management/v2#!/Anomaly/delete_ips_by_id

I think its fine to get this out the door if we're in a hurry, but this should be added asap in my opinion.